bin/package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "@glimmer-workspace/bin",
+  "type": "module",
   "private": true,
+  "repo-meta": {
```

Comment:
This PR  introduces a way of specifying package-specific tweaks to the overall linting
configuration. The details are documented in @glimmer-workspace/eslint-plugin/README.md